src/components/GestureDiagram.tsx

```diff
@@ -243,7 +243,17 @@ const GestureDiagram = ({
         const { x, y } = positions[i]
         return (
           <path
-            d={`M ${x} ${y} l ${segment.dx} ${segment.dy}`}
+            d={
+              path === 'rdld'
```

Comment:
An inline comment would help clarify what's going on here, as it might not be completely obvious to other developers.